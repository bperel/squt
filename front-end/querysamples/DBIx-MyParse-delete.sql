DELETE LOW_PRIORITY QUICK IGNORE
FROM test.table_name
WHERE 1
ORDER BY column_name
LIMIT 1234
