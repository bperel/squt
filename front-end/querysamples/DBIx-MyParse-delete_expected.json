{"error":"for performance and security reasons, squt does not allow queries longer than 2000 characters"}
